OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/CanonicalizedSecretKey.java

```diff
@@ -319,6 +320,28 @@ public RSAPrivateCrtKey getCrtSecretKey() throws PgpGeneralException {
         return (RSAPrivateCrtKey)retVal;
     }
 
+    // For use only in card export; returns the secret key.
```

Comment:
Same comment applies here as to the CanonicalizedPublicKey. Maybe move the smart card related methods to a new file containing the CanonicalizedSecretKey